### PR TITLE
1690: put visualization component into draggable panel

### PR DESCRIPTION
**Pull request by adnauseum**
**Changes**:
- Rename `FloatingVizManagement` to `MapVizManagement`
- Add styling & behavior tweaks from Map UX meeting.
- Add grouped visualization picker: `<NewVisualizationPickerGrouped />`
- Render visualizations in a draggable panel.
- Reactor styling & behavior of visualization list.

**Before**:

image

**After**: 

https://user-images.githubusercontent.com/24446573/230714117-1daebbc4-78cf-471b-b277-bd5e27f51e73.mp4

### CODE REVIEW

**Comment by bobular** [comment]
Looks great!  Just a note about the title.

The floating viz panel is not a "Configure XYZ" panel. It's a "XYZ" panel.  I know we may have ambitions to have a multi-step plot configuration that saves on screen space, but this would be the topic of another Map UX call (maybe tomorrow). Also, implementing that isn't trivial at the moment.

I would just make the floating plot draggable big enough to contain most of the plot for now, and we focus on wiring up other things and come back to optimizing the footprint of the floaters ASAP.

File: packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx [hunk 1]
```diff
@@ -1,28 +1,28 @@
-import { CSSProperties, ReactNode, useEffect, useState } from "react";
```
Comment (adnauseum) [inline packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx:1]: 🤔 I wonder if my editor config is fighting with some other config...

Comment (adnauseum) [reply packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx:1]: Ignore this file for now, I suppose.

Comment (bobular) [reply packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx:1]: After every commit the code should be going through `prettier` so our editors' quirks shouldn't matter. However, during the monorepo migration, something happened in this arena...

**Review by bobular — COMMENTED** [review]
The draggable floating viz container is great for now.  Obviously longer term we have to save space and not make the user scroll down (which will require a bigger/wider refactoring), but for this issue/PR it is great.

To wrap up the parent issue - wait which issue is that? - ah [here it is](https://github.com/VEuPathDB/web-eda/issues/1499) - could we just tidy up the list of existing visualizations (with some "active status" styling) and make the new viz icons fit on the screen? (see my screenshot below) I would remove the "Unnamed computation (pass)" heading possibly get rid of the `[+ new plot]` button as previously discussed, but I'll leave that up to you. Thanks!

image

(damn - I did all this yesterday and forgot to submit it)

File: packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx [hunk 2]
```diff
@@ -368,7 +368,7 @@ function MapAnalysisImpl(props: Props & CompleteAppState) {
         icon: <BarChartSharp />,
         renderWithApp: (app) => {
           return (
-            <FloatingVizManagement
+            <MapVizManagement
```
[the remark this answers is not in the document]
Comment (adnauseum) [reply packages/libs/eda/src/lib/map/analysis/MapAnalysis.tsx:371]: Thanks for catching that! Interesting behavior on VS Code's part. I went to <kbd>rename file</kbd> and it did a copy and kept the originals!

I deleted the originals in: https://github.com/VEuPathDB/web-monorepo/pull/31/commits/a99ff498c7991bca9e9602940689c82ae85ea633

File: packages/libs/eda/src/lib/core/components/visualizations/VisualizationsContainer.tsx [hunk 3]
```diff
@@ -871,3 +971,47 @@ function deleteComputationWithNoVisualizations(
     ]);
   }
 }
+
+type VisualizationGroup = {
+  groupName: string;
+  groupDescription: string;
+  groupVisualizationsList: VisualizationOverview[];
+};
+function groupVisualizations(
```
Comment (adnauseum) [inline packages/libs/eda/src/lib/core/components/visualizations/VisualizationsContainer.tsx:980]: Danielle mentioned that she can do this on the backend soon!

File: packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx (hunk 1 above)
Comment (adnauseum) [reply packages/libs/coreui/src/components/containers/DraggablePanel/index.tsx:1]: > I think CoreUI had a different config

Ahh, that makes total sense! We could do a massive formatting pr in the monorepo!

File: packages/libs/eda/src/lib/map/index.ts [hunk 4]
```diff
@@ -12,4 +12,4 @@ export type SiteInformationProps = {
 export const mapNavigationBackgroundColor = 'white';
 export const mapNavigationBorder: CSSProperties['border'] = '1px solid #D9D9D9';
 export const mapSideNavigationActiveMenuItemBackground: CSSProperties['background'] =
-  '#E1F5FF';
+  '#ECF7F9';
```
Comment (dmfalke) [inline packages/libs/eda/src/lib/map/index.ts:15]: **Question:** should this value come from the theme palette? Not sure if we have something comparable.

**Review by dmfalke — APPROVED** [review]
This all looks great to me! See my question below. Otherwise, I think this is good to go.

File: packages/libs/eda/src/lib/map/index.ts (hunk 4 above)
Comment (adnauseum) [reply packages/libs/eda/src/lib/map/index.ts]: Whoops! I forgot to delete that! It was a dangler placeholder!! The component uses the theme hook!

https://github.com/VEuPathDB/web-monorepo/pull/31/commits/3a86b1c56dca486bc86d3fc28de4e41b0ff06834

File: packages/libs/eda/src/lib/core/components/visualizations/VisualizationsContainer.tsx (hunk 3 above)
Comment (adnauseum) [reply packages/libs/eda/src/lib/core/components/visualizations/VisualizationsContainer.tsx:980]: https://github.com/VEuPathDB/EdaDataService/pull/258

**Pull request merged**